test/unit/chromic_pdf/pdf/connection_test.exs

```diff
@@ -29,6 +29,12 @@ defmodule ChromicPDF.ConnectionTest do
       expect(ChromeMock, :spawn, fn ^opts -> {:ok, @port} end)
       assert init({self(), opts}) == {:ok, new_state()}
     end
+
+    test "it spawns custom Chrome executeable and initializes its state" do
+      opts = [discard_stderr: false, no_sandbox: true, chrome_executeable: "/cutom/chrome", chrome_args: "--foo"]
```

Comment:
Maybe let's just add this `chrome_executable` option to the test above. These unit tests are a bit pointless in general, they mostly assert that the args are passed to `ChromeImpl` (i.e. the `Keyword.take` call).

Reply:
Hi,

thank you for your quick response 👍
I implemented the requested changes.

Thanks and have a great weekend.